Discoteque.API/Program.cs

```diff
@@ -58,70 +69,70 @@ async void PopulateDb(WebApplication app)
         var tourService = scope.ServiceProvider.GetRequiredService<ITourService>();
 
         // Artists
-        await artistService.CreateArtist(new Discoteque.Domain.Models.Artist
```

Comment:
This db population can be made in another class there are too many lines of code here